### DISCUSSION TITLE
documentation or goodfirst issue

### DISCUSSION THREAD

**Discussion by mehrdadbn9** [aquasecurity/tracee#4131] [Development]
Hi
 is it possible to help me as newcomer to participate in this community with writing docs or good first issues?

**Comment by geyslan** [comment]
@mehrdadbn9 hey, thanks for considering to contribute to Tracee.

You might track the label [good first issue](https://github.com/aquasecurity/tracee/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22). Anything, let us know.
